examples/js/exchange-capabilities.js

```diff
@@ -20,15 +20,32 @@ const isWindows = process.platform == 'win32' // fix for windows, as it doesn't
     let inexistentApi = 0
     let implemented = 0
     let emulated = 0
+ 
```

Comment:
Can you remove this

Reply:
Which specific code to remove in this PR do you mean? All of it?